nepalingo-web/src/components/randomQuotes.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { useLanguage } from '../hooks/useLanguage';
+import useQuotes from '../hooks/useQuotes';
+
+const RandomQuoteComponent: React.FC = () => {
+    const { language } = useLanguage();
+    const { randomQuote } = useQuotes({ language });
+
+    if (!randomQuote) {
+        return <div>Loading...</div>;
+    }
+    const { text, } = randomQuote;
+    const translatedText = [language];
```

Comment:
wouldn't this just show the current language selected? I think you need to get the `translation` from the `randomQuote`